src/components/UrlUtilsService.js

```diff
@@ -104,9 +104,9 @@ goog.provide('ga_urlutils_service');
         // Remove proxy from the URL
         this.unProxifyUrl = function(url) {
           if (this.isValid(url)) {
-            var reg = new RegExp(['^(http|https):\/\/(service-proxy.',
-                '(dev|int|prod).bgdi.ch|proxy.geo.admin.ch)',
-                '\/(http|https)\/(.*)'].join(''));
+            var reg = new RegExp(['^(http|https)://(service-proxy.',
```

Comment:
When testing in console, it seems like constructor is taking care of it. Anyhow, it might be confusing, since most of our regex is tested using regex101.